web/Web/web/static/js/document-viewer.js

```diff
@@ -958,6 +1021,19 @@ docView.prototype = {
         }
       }
 
+      if (options.reviewMode) {
```

Comment:
The bolding feature is useful, so it shouldn't be just for the review interface only.

Reply:
This whole code segment can be moved into `viewPreviouslyJudgedDocument` function. You can add a new parameter that refers to the div element that we want to bold `$(this)`, and bold it right away without having to go through an `each` loop.

Reply:
Got rid of the loop. 
If I move the segment into viewPreviouslyJudgedDocument, the first time you go to Review, the current doc will not be bolded.

viewPreviouslyJudgedDocument is only called when you explicitly click those documents in the review list.